### PR TITLE
added license and resolve object_name display

### PR DESCRIPTION
**Pull request by gjlawran**
Added license and object name display per JIRA656

### CODE REVIEW

File: ckanext/bcgov/templates/package/read.html [hunk 1]
```diff
@@ -119,7 +119,7 @@
   {% block package_additional_info %}
     <h3>Additional Information</h3>
     <div id="additional-information">
-      {% if pkg.object_name and pkg.resources and pkg.resources[0].resource_storage_location == 'BCGW Data Store' %}
+      {% if pkg.object_name and pkg.resources %}
```
Comment (wardi) [inline ckanext/bcgov/templates/package/read.html]: So, this is the change that should be visible. I'll find a dataset with a non-"BCGW Data Store" resource[0] and see if the change takes place.

Comment (gjlawran) [inline ckanext/bcgov/templates/package/read.html]: Yes - that is correct. If you put the object_name into a geographic pacakge
it should display it as long as their is a data resource of one kind or
another.

Comment (gjlawran) [inline ckanext/bcgov/templates/package/read.html]: Seems to have worked!

Comment (wardi) [inline ckanext/bcgov/templates/package/read.html]: yes, it was quick and easy. I'm just getting set up with a local dev environment so I can make and test the other little changes.

**Pull request merged**